ttnn/cpp/ttnn/operations/creation.hpp

```diff
@@ -384,14 +452,13 @@ struct Arange {
 }  // namespace creation
 }  // namespace operations
 
-constexpr auto full =
-    ttnn::decorators::register_operation_with_auto_launch_op<"ttnn::full", ttnn::operations::creation::Full>();
+constexpr auto full = ttnn::decorators::register_operation<"ttnn::full", ttnn::operations::creation::Full>();
```

Comment:
Here and below -- these functions don't need to be wrapped with auto launch, but the reason I removed them in this PR is https://github.com/tenstorrent/tt-metal/issues/15442.